sample.go

```diff
@@ -12,14 +12,17 @@ import (
 // traceID.
 type Sampler func(id uint64) bool
 
-func neverSample(_ uint64) bool { return false }
+// NeverSample samples no traces
+func NeverSample(_ uint64) bool { return false }
 
-func alwaysSample(_ uint64) bool { return true }
+// AlwaysSample samples all traces, it is appropriate for easy prototyping but not
+// meant to be used in production.
```

Comment:
AlwaysSample will always return true. If used by a service it will always start traces if no upstream trace has been propagated. If an incoming upstream trace is not sampled the service will adhere to this and only propagate the context.